packages/scanner-global-library/src/page-handler.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { inject, injectable } from 'inversify';
+import { GlobalLogger } from 'logger';
+import { Page } from 'puppeteer';
+
+@injectable()
+export class PageHandler {
```

Comment:
is this class used anywhere?

Reply:
Correct, I'll add in next PR to unblock this one